src/routes/(admin)/dashboard/Applyform.svelte

```
@@ -0,0 +1,101 @@
+	{#each test_summary_box as stat}
+		<div
+			class="m-4 w-1/4 rounded-xl bg-white p-4 hover:scale-105 hover:shadow-xl"
+			style="flex: 0 0 20%;"
+		>
+			<div class="flex items-center gap-2 font-semibold">
+				<p class="mb-2 border-b">{stat.title}</p>
+			</div>
+			<p class="text-sm">{stat.description}</p>
+			<div class="p-12 text-5xl">{stat.count}</div>
+		</div>
+	{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)